scripts/start/readCheckScenarioConfig.R

```diff
@@ -22,7 +22,7 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
     cfg <- gms::readDefaultConfig(remindPath)
   }
   scenConf <- read.csv2(filename, stringsAsFactors = FALSE, na.strings = "", comment.char = "#",
-                                  strip.white = TRUE, blank.lines.skip = TRUE)
+                                  strip.white = TRUE, blank.lines.skip = TRUE, check.names = FALSE)
```

Comment:
if we do not check names, do we miss some important checks?

Reply:
I dont think so. We need this to avoid the dollar signs (from the switch names) beeing replaced with dots.